website/docs/cloud-docs/agents/telemetry.mdx

```diff
@@ -4,40 +4,46 @@ page_title: Telemetry - Terraform Cloud Agents - Terraform Cloud and Terraform E
 
 # Telemetry
 
-Terraform Cloud Agents emit telemetry data using the [OpenTelemetry](https://opentelemetry.io/) protocol.
+Terraform Cloud Agents may be configured to emit helpful telemetry data,
+including both trace spans and metrics.
 
-## Configuration
+## Telemetry architecture
 
-To configure your agent to emit telemetry data, you must include the `-otlp-address` flag or `TFC_AGENT_OTLP_ADDRESS` environment variable. This should be set to the host:port address of an [OpenTelemetry collector](https://opentelemetry.io/docs/concepts/data-collection/). This address should be a GRPC server running an OLTP collector.
+Terraform Cloud Agents emit telemetry data using the
+[OpenTelemetry](https://opentelemetry.io/) protocol. The agent is a simple
+producer of telemetry data, and relies on an OpenTelemetry Collector to perform
+aggregation, post-processing, and exporting of telemetry data to upstream
+technology and/or vendors. OpenTelemetry is an extensible telemetry protocol and
+open source platform, which enables the Terraform Cloud Agent to remain
+vendor-agnostic while producing high-value telemetry data for a wide array
+of platforms.
 
-Optionally, you can pass the `-otlp-cert-file` or `TFC_AGENT_OTLP_CERT_FILE`. The agent will use a certificate at the path supplied to encrypt the client connection to the OpenTelemetry collector. When omitted, client connections are not secured.
+![telemetry architecture diagram](../../../img/teletry-arch.svg)
```

Comment:
This link is broken. I think there is one too many `../`

Reply:
Woops, it would also help if I spelled `telemetry` correctly 🤦‍♂️ Thanks!